src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.targets

```diff
@@ -47,6 +47,9 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeLibrary Condition="'$(IlcMultiModule)' == 'true'" Include="$(SharedLibrary)" />
       <NativeLibrary Include="$(IlcSdkPath)$(StandaloneGCSupportName)$(LibrarySuffix)" />
       <NativeLibrary Include="$(IlcSdkPath)zlibstatic$(LibFileExt)" />
+      <NativeLibrary Include="$(IlcSdkPath)brotlicommon$(LibFileExt)" />
+      <NativeLibrary Include="$(IlcSdkPath)brotlienc$(LibFileExt)" />
+      <NativeLibrary Include="$(IlcSdkPath)brotlidec$(LibFileExt)" />
```

Comment:
When we build .NET with system brotli, we don't include broticommon:

https://github.com/dotnet/runtime/blob/f0405a977ebf5d430ad73e57e52ad5cf5297eff6/src/native/libs/System.IO.Compression.Native/extra_libs.cmake#L23-L28

Reply:
When building against system brotli, brotlicommon is a private dependency of the brotlienc and brotlidec shared libraries, so it can be omitted. When statically linking, we need to specify all three.